nbautoexport/sentinel.py

```diff
@@ -50,6 +50,20 @@ class NbAutoexportConfig(BaseModel):
     class Config:
         extra = "forbid"
 
+    # deprecated in pydantic v2.0
```

Comment:
I don't think "deprecated" is correct. This should say "renamed".

Reply:
They do still work, but have a deprecation warning:

```
PydanticDeprecatedSince20: The `dict` method is deprecated; use `model_dump` instead. Deprecated in Pydantic V2.0 to be removed in V3.0. See Pydantic V2 Migration Guide at https://errors.pydantic.dev/2.0.3/migration/
```

Reply:
Ah, you're right. That's my mistake.